specification/mediaservices/resource-manager/Microsoft.Media/stable/2018-07-01/Encoding.json

```diff
@@ -1242,7 +1129,7 @@
         }
       },
       "type": "object",
-      "description": "Describes a list of inputs to a Job."
+      "description": "Describes a list of of inputs to a Job."
```

Comment:
I'm seeing a few typos, such as `of of`, `the the`, `modifed`, are they intentional?